### ISSUE TITLE
AAS-Server: files not served in 1.4.0 (regression from 1.3.3)

### ISSUE THREAD

**Issue by StenGruener** [eclipse-basyx/basyx-java-components#329]
Hi,

i am trying to access a file embedded in loaded aasx package.

1.3.1:
![image](https://user-images.githubusercontent.com/69786685/234491278-b33d06c7-0860-4d8d-8759-ce4d65d0f95d.png)
file is accessible
host:8081/aasServer/files/aasx/test.txt

1.4.0:
Json is same, but file is not served
![image](https://user-images.githubusercontent.com/69786685/234492059-03866318-a3f1-4e7b-a502-ceb87c8e5eae.png)

Pulling docker images from docker hub.

AAS is attached
[EmptyAAS.zip](https://github.com/eclipse-basyx/basyx-java-components/files/11329845/EmptyAAS.zip)

**Comment by FrankSchnicke** [comment]
Thanks for pointing this out. We will take a look at it and come back with a solution.

**Comment by mdanish98** [comment]
Hi @StenGruener 

This issue was due to the recent unpacking changes. The docbase path was mismatched.

We fixed this issue and will be included in the **next release**.
Fix-related PRs are merged in the development branch of both SDK and Components, feel free to check it.

Now the files will be accessed as before except that there is a slight change in the URL as highlighted below:
![image](https://github.com/eclipse-basyx/basyx-java-components/assets/62088117/d0a364a0-bca0-40e1-af05-f9b25064fbfd)


File view:
![image](https://github.com/eclipse-basyx/basyx-java-components/assets/62088117/44d850d3-b9c9-4931-a308-34bec2722123)
